addon/components/models-table.js

```diff
@@ -982,7 +984,6 @@ export default Component.extend({
    * @name ModelsTable#setup
    */
   setup: on('init', function() {
-    this._setupSelectedRows();
     this._setupExpandedRows();
```

Comment:
I wonder why they did it this way when columns are set directly instead of this setup step. Maybe the assume you want to reset selected items every time it's loaded?